README.md

```diff
@@ -224,31 +224,15 @@ docker build -t retdec:dev . -f Dockerfile.dev
 ```
 
 ### Run Container
-
-To decompile a binary, create a container to upload the binary to:
-```
-docker create --name retdec_init retdec
-```
-
-Upload the binary (note the destination directory should be a directory with read/write permissions, such as `/home/retdec/`):
-```
-docker cp <file> retdec_init:/destination/path/of/binary
+Make sure the folder writable by retdec
```

Comment:
I am wondering: Why is this step necessary? I was able to run a decompilation even without it (the produced files were created under the user that ran the container). Wouldn't it suffice for the directory to be readable and writeable for the current user?

Reply:
I see. Thank you for the explanation!